assets/js/llms-metabox-voucher.js

```diff
@@ -138,15 +138,23 @@
 	 *
 	 * @since Unknown
 	 * @since 5.9.0 Add nonce.
+	 * @since [version] Add check for empty vouchers.
 	 *
 	 * @return {void}
 	 */
 	function check_voucher_duplicate() {
 
+		var vouchers = get_codes_from_inputs();
+
+		if( ! vouchers.length ) {
+			$( "#post" ).submit();
+			return;
+		}
+
 		var data = {
-			action: 'check_voucher_duplicate', 'postId' :
-			jQuery( '#post_ID' ).val(),
-			'codes' :  get_codes_from_inputs(),
+			action: 'check_voucher_duplicate',
+			postId: $( '#post_ID' ).val(),
+			codes:  vouchers,
```

Comment:
```suggestion
			action: 'check_voucher_duplicate',
			postId: $( '#post_ID' ).val(),
			codes: vouchers,
```
can you just fix this old formatting issue too? <3